### ISSUE TITLE
Brew install not working on Monterey 12.3

### ISSUE THREAD

**Issue by holleman** [cgascoig/isctl#37]
(venv) KEITHH-M-RMYN:src keithh$ brew --version
Homebrew 3.4.3
Homebrew/homebrew-core (git revision a57b31329bc; last commit 2022-03-24)
Homebrew/homebrew-cask (git revision c38da9fc2e; last commit 2022-03-24)
(venv) KEITHH-M-RMYN:src keithh$ brew install cgascoig/isctl/isctl
==> Tapping cgascoig/isctl
Cloning into '/usr/local/Homebrew/Library/Taps/cgascoig/homebrew-isctl'...
remote: Enumerating objects: 24, done.
remote: Counting objects: 100% (24/24), done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 24 (delta 6), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (24/24), done.
Resolving deltas: 100% (6/6), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/cgascoig/homebrew-isctl/isctl.rb
isctl: Calling bottle :unneeded is disabled! There is no replacement.
Please report this issue to the cgascoig/isctl tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/cgascoig/homebrew-isctl/isctl.rb:9

Error: Cannot tap cgascoig/isctl: invalid syntax in tap!
(venv) KEITHH-M-RMYN:src keithh$

![Screen Shot 2022-03-24 at 10 39 33 AM](https://user-images.githubusercontent.com/5355896/159981279-7e87730b-ff9f-41d1-b192-08cffb82c17f.png)

**Comment by cgascoig** [comment]
Thanks for catching this. Should be fixed now in the homebrew formula (https://github.com/cgascoig/homebrew-isctl/blob/master/isctl.rb). Please give it another try and reopen this issue if it is not fixed.

**Comment by holleman** [comment]
Got it installed but this one tripped me up for a sec - expected?  Want to call it out in the README?

KEITHH-M-RMYN:~ keithh$ brew install cgascoig/isctl/isctl
==> Tapping cgascoig/isctl
Cloning into '/usr/local/Homebrew/Library/Taps/cgascoig/homebrew-isctl'...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (27/27), done.
remote: Compressing objects: 100% (25/25), done.
remote: Total 27 (delta 7), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (27/27), 4.05 KiB | 2.02 MiB/s, done.
Resolving deltas: 100% (7/7), done.
Tapped 1 formula (13 files, 9.3KB).
==> Downloading https://github.com/cgascoig/isctl/releases/download/v0.1.5/isctl_0.1.5_Darwin_x86_64.zip
==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/269819207/d7154380-e18f-11eb-8553-eead2e6a4bd3?X-Amz-Algorithm=AWS4
######################################################################## 100.0%
==> Installing isctl from cgascoig/isctl
Error: The following formula cannot be installed from bottle and must be
built from source.
  isctl
Install the Command Line Tools:
  xcode-select --install

KEITHH-M-RMYN:~ keithh$

**Comment by cgascoig** [comment]
Hmm .. no, not expected. 

I don't usually hand craft/edit the homebrew formula - I used GoReleaser to generate it. I need to update to the latest API spec anyway so let me do that and create a new release with GoReleaser which should hopefully fix the brew formula

**Comment by cgascoig** [comment]
OK, I've just released 0.1.6 with a fresh homebrew formula from goreleaser ... hopefully should be good now.

**Comment by holleman** [comment]
Great thanks - it works.  Was looking to update in order to add this but you took care of it.

```
KEITHH-M-RMYN:~ keithh$ isctl get virtualization iwecluster --jsonpath '$.Name'
ucsp598
KEITHH-M-RMYN:~ keithh$
```
